src/build.sh

```diff
@@ -147,23 +164,91 @@ function parse_build_options()
         shift
         ;;
       --menu | -n)
-        options_values['MENU_CONFIG']="${configurations[menu_config]:-nconfig}"
+        options_values['MENU_CONFIG']="${configurations[menu_config]:-"nconfig"}"
         shift
         ;;
       --doc | -d)
-        options_values['DOC_TYPE']="${configurations[doc_type]:-htmldocs}"
+        options_values['DOC_TYPE']="${configurations[doc_type]:-"htmldocs"}"
         shift
         ;;
+      --pkg | -p)
+        options_values['PKG']="$2" # Default value is debian
+        shift 2
+        ;;
       --)
         shift
         ;;
       *)
-        complain "Invalid option: $option"
-        exit 22 # EINVAL
+        complain "Invalid option: $1"
+        return 22 # EINVAL
+        shift
         ;;
     esac
   done
 
+  if [[ "${options_values['PKG']}" == 'no' ]]; then
+    options_values['PKG']=''
+  fi
+}
+
+# This function returns the makefile rule for creating a package, according to
+# the type of package required
+#
+# @pkg_type The type of package created
+#
+function pkg_get_rule()
+{
+  local pkg_type="$1"
+
+  case "$pkg_type" in
+    debian)
+      echo 'bindeb-pkg'
+      return
+      ;;
+    *)
+      return 95 # ENOTSUP
+      ;;
+  esac
+}
+
+# This function prints information about recently created debian packages
+#
+# @pkg_type The type of package created. If empty, nothing is done.
+# @revision The revision number of the package
+#
+function pkg_info()
+{
+  local pkg_type="$1"
+  local revision="$2"
+
+  case "$pkg_type" in
+    debian)
+      pkg_info_debian "$revision"
+      return
+      ;;
+  esac
+}
+
+# This function prints information about recently created debian packages
+#
+# @revision The revision number of the package
+#
+function pkg_info_debian()
```

Comment:
how about use `pkg_info_deb`? I think this is a better name because we can use the deb package for different OSes based on Debian.